book/3-section-III/3.09-chap-9/3.09.03-continuous-improvement.adoc

```diff
@@ -33,7 +33,7 @@ The scope of the word "process" is immense. Examples include:
 * The general steps followed in creating and executing a project
 * The overall flow of work in software development, from idea to operation
 
-This breadth of usage requires us to be specific in any discussion of the word "process." In particular, we need to be careful in understanding the concepts of efficiency, variation, and effectiveness. These concepts lie at the heart of understanding process control and improvement and how to correctly apply it in the digital economy.
```

Comment:
Punctuation  belongs inside the quote. Doing this American style. http://blog.apastyle.org/apastyle/2011/08/punctuating-around-quotation-marks.html